languages/go/reference/README.md

```diff
@@ -27,12 +40,19 @@
 - [Boolean logic](../../../reference/concepts/boolean_logic.md)
 - Channels
 - Code Formatting
+- context.Context
 - [Comments](../../../reference/concepts/comments.md)
 - Concurrency
 - [Conditionals](../../../reference/concepts/conditionals.md)
+- CSV Handling
+- Debugging
 - Defer
 - Errors and Error Handling
 - [Functions](../../../reference/concepts/functions.md)
+- Globals
+- Ignoring (return) values
+- Init function
+- JSON Handling
```

Comment:
Is this a core Go (language) feature? I can imagine it being so, but I don't know.

Reply:
Since this is a stumbling block for many newcomers to Go, I think we should have an exercise on it.

Reply:
Right. I feel this is at the edge of what is a Concept Exercise and what is a Practice Exercise.  @iHiD What do you think?

Reply:
More information: In Go json is a bit special. It is the only time most people will ever use `tags` in Go:

```go
type Info struct{
    Name string `json:"name"`
}
```

Also you need to understand, that only exported fields can ever be used for json. That's why I'd vote for calling this a concept in Go.

Reply:
Agreed. The `tags` feature alone makes this worthy of inclusion.